modules/apps/liferay-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,514 @@
+	margin: 0;
+	padding: 0;
+}
+
+body {
+	font-family: Helvetica, Arial, sans-serif;
+	height: 100%;
+	margin: 0;
+	padding: 0;
+}
+
+.body-content {
+	height: 100%;
+	left: 0px;
+	position: absolute;
+	top: 0px;
+	width: 100%;
+	z-index: -3;
+}
+
+#container {
+	min-height: 100%;
+	position: relative;
+}
+
+.portlet-decorate .portlet-content {
+	border-style: none;
+}
+
+.portlet-title-text {
+	display: none;
+}
+
+.portlet-content, .portlet-minimized .portlet-content-container {
+	padding:0px;
+}
+
+.portlet-body {
+	margin: 0 auto;
+}
+
+.sign-in {
+	border: 2px solid #FFFFFF;
+	border-radius: 200px;
+	color: #FFFFFF;
+	display: block;
+	float: right;
+	margin-right: 15px;
+	padding: 6px;
+	text-transform: uppercase;
+}
+
+.header-banner {
+	color: white;
+	left: 55px;
+	position: absolute;
+	top: 260%;
+}
+
+@media only screen and (max-width: 650px) {
+	.site-title {
+		margin-top: -40px;
+		margin-left: -10px;
+	}
+
+	.navbar-brand {
+		font-size: 15px !important;
+		margin-top: 6px !important;
+	}
+	.navigation-bar ul {
+		margin-left: 10px;
+
+		li {
+			font-size: 12px;
+			width: 90px;
+		}
+	}
+
+	.navbar-toggle {
+		height: 25px;
+		margin-left: 0px;
+	}
+
+	.team-text {
+		font-size: .8em !important;
+		overflow: overlay;
+		width: 100% !important;
+	}
+
+	.architecture-wrapper2 {
+		display: inline-block !important;
+	}
+}
+
+@media only screen and (max-width: 766px) {
+	.navigation-bar {
+		display: inline-block;
+		float: none;
+		vertical-align: top;
+	}
+
+	.navigation-bar .navbar-collapse {
+		text-align: center;
+	}
+
+	.navigation-bar ul {
+		display: inline-block;
+		margin-left: 5px;
+		margin-top: 5%;
+
+		li {
+			display: inline;
+		}
+
+		a {
+			background-color: rgba(0, 0, 0, 0.45);
+			border-radius: 200px;
+			margin-bottom: -10px;
+		}
+	}
+}
+
+@media only screen and (max-width: 1200px) {
+	.Image2Text {
+		font-size: 1em !important;
+		height: 50% !important;
+		margin: 1em !important;
+		margin-top: 7% !important;
+		max-width: none !important;
+		width: auto;
+	}
+}
+
+@media only screen and (max-width: 1745px) {
+	.Image2Text {
+		margin: 8em !important;
+	}
+}
+
+@media only screen and (max-width: 1450px) {
+	.Image2Text {
+		margin: 6em !important;
+	}
+}
+
+@import url('https://fonts.googleapis.com/css?family=Oswald:300');
+
+h1 {
+	font-family: 'Oswald', Helvetica, sans-serif;
+	font-size: 4.5vw;
+	letter-spacing: 3px;
+	margin-top: 20px;
+	margin-left: 6px;
+	text-align: left;
+	text-transform: uppercase;
+	white-space: nowrap;
+}
+
+img {
+	position: relative;
+	width: 100%;
+}
+
+h2 {
+	left: 200px;
+	position: absolute;
+	top: 500px;
+	width: 50%;
+
+	span {
+		color: white;
+		font-style: italic;
+		font-family: avenir;
+		padding: 10px;
+	}
+}
+
+.beginning {
+	padding: 10px;
+	position: relative;
+	text-align: center;
+}
+
+.navbar-brand {
+	float: right;
+	font-size: 18px;
+	margin-top: 9px;
+	position: relative;
+	text-transform: uppercase;
+	-webkit-animation: colorchange 20s infinite alternate;
```

Comment:
vendor prefix's should be listed alphabetically as if the vendor prefix did not exist.